src/main/java/mateusz/michal/chat/Service/RegistrationService.java

```diff
@@ -42,6 +46,10 @@ private boolean isNameNotPresent(String name){
         return name.equals("");
     }
 
+    private boolean isNameIncorrect(String name){
+        return (name.startsWith(" ") || name.endsWith(" ") || isNameHasSpecialCharacters(name));
```

Comment:
nit: metoda mogłaby się nazywać tak (krócej, bardziej poprawnie ;))
```suggestion
        return (name.startsWith(" ") || name.endsWith(" ") || hasNameSpecialCharacters(name));
```